src/pages/account/your-beacon-registry-account.tsx

```diff
@@ -1,16 +1,24 @@
-import { GetServerSideProps, GetServerSidePropsContext } from "next";
+import { GetServerSideProps } from "next";
 import React, { FunctionComponent } from "react";
 import { Grid } from "../../components/Grid";
 import { Layout } from "../../components/Layout";
 import { BeaconRegistryContactInfo } from "../../components/Mca";
 import { PageHeading, SectionHeading } from "../../components/Typography";
+import { IAccountHolderDetails } from "../../entities/accountHolderDetails";
+import { IBeacon } from "../../entities/beacon";
+import {
+  BeaconsGetServerSidePropsContext,
+  withContainer,
+} from "../../lib/container";
 
 interface YourBeaconRegistyAccountPageProps {
   id?: string;
+  accountHolderDetails: IAccountHolderDetails;
+  beacons: IBeacon[];
 }
 
 export const YourBeaconRegistyAccount: FunctionComponent<YourBeaconRegistyAccountPageProps> =
-  (): JSX.Element => {
+  (props: YourBeaconRegistyAccountPageProps): JSX.Element => {
```

Comment:
@tm1303 here you can use [object deconstruction](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring) to remove the need to have the props which will help with readability.

This can then become:
```suggestion
  ({accountHolderDetails, beacons}: YourBeaconRegistyAccountPageProps): JSX.Element => {
```